coretex/cli/project.py

```diff
@@ -0,0 +1,54 @@
+from typing import Optional
+
+import click
+from ..configuration import loadConfig, saveConfig
+from .. import Project, ProjectType
+
+
+def createProject(name: str) -> None:
+    try:
+        new_project = Project.createProject(name, ProjectType.other)
```

Comment:
Just `project`. User should pick the type of the project which is to be created, it should not be hardcoded to `other`